pages/blog/google-season-of-docs-2022.md

```diff
@@ -0,0 +1,95 @@
+
+
+## 📑 About our Docs project
+
+### 🔎 Our current Docs problem
+Our current Docs and their Information Architecture (IA) needs a major makeover. The current content buckets are far from ideal and much basic content is missing to help onboard new contributors. Users new to our API spec need `/Conceptual` docs that explain our spec terminology in more detail with engineering diagrams: people often learn visually! We also have to move our CLI docs under the Docs upcoming new `Reference` content bucket; currently we have a README version of CLI docs only. Similarly, we're adding a new and broader `/Tools` section of documentation for our tools in individual tools' GitHub repos, under a `/docs` directory. Those should still remain there and continue to be maintained, but they also need to be documented in our Docs in a less informal way than what you see in a README. In time, we also need to add many more tutorials (i.e. Websocket, Kafka, etc) and Usecases and Troubleshooting Guides, under a new `How-To` section.
+
+
+### 🎯 Our Docs project’s scope
+We're already invested in utilizing the [Diátaxis methodology](https://diataxis.fr/) for determining our **content buckets** _(Concepts, Tutorials, Tools, How-To Guides, Reference)_, it makes sense to ADD NEW landing pages to introduce each content bucket. In addition, each content bucket landing page could include cards featuring content the community has already requested but that we still need contributions for. For example, instead of having a card that says "coming soon," we would have the cards say, "Contributors Needed." Lastly, we have several CLI and Tools markdown README documentation in miscellaneous GitHub repos that we're in the process of migrating over to the main Docs site section. This last item would be another task we'd incoropate as part of our goal to finalize our 2022 AsyncAPI Docs Information Architecture makeover, which is not only mentioned in one of our OSS blog posts titled ["Change is coming to our AsyncAPI Developer Documentation"](https://www.asyncapi.com/blog/changes-coming-docs), but is also extensively documented in our [AsyncAPI Docs GitHub Project Board](https://github.com/orgs/asyncapi/projects/8). 
```

Comment:
`We're already invested` should be `We already invested` I think

Reply:
```suggestion
We're already invested in utilizing the [Diátaxis methodology](https://diataxis.fr/) for determining our **content buckets** _(Concepts, Tutorials, Tools, How-To Guides, Reference)_, it makes sense to **add new** landing pages to introduce each content bucket. In addition, each content bucket landing page could include cards featuring content the community has already requested but that we still need contributions for. For example, instead of having a card that says "coming soon," we would have the cards say, "Contributors Needed." Lastly, we have several CLI and Tools markdown README documentation in miscellaneous GitHub repos that we're in the process of migrating over to the main Docs site section. This last item would be another task we'd incorporate as part of our goal to finalize our 2022 AsyncAPI Docs Information Architecture makeover, which is not only mentioned in one of our OSS blog posts titled ["Change is coming to our AsyncAPI Developer Documentation"](https://www.asyncapi.com/blog/changes-coming-docs) but is also extensively documented in our [AsyncAPI Docs GitHub Project Board](https://github.com/orgs/asyncapi/projects/8). 
```

Reply:
This paragraph is too long. I suggest making it shorter also probably by making sentences shorter.

Reply:
> `We're already invested` should be `We already invested` I think

No, that would give it a different meaning :)

Reply:
> This paragraph is too long. I suggest making it shorter also probably by making sentences shorter.

sure!